release_build_files/readme.md

```diff
@@ -567,6 +567,11 @@ code.
 
 ## Release Notes
 
+### 8.1.0
+-   Changes
+    -   Firestore: Fixed a linker error when DocumentChange::npos was used.
+        ([#474](https://github.com/firebase/firebase-cpp-sdk/pull/474)).
```

Comment:
Optional: I think the changelog usually links to issues, not PRs, so this link could probably be omited.

Reply:
It looks like there is a mixture of issues and PRs linked to in previous change log entries. I see your point but I personally feel like adding a link to additional context is useful for future readers. If this issue had been discovered externally then there would be an issue number; however, since it was discovered by me, a developer on the Firestore team, no issue was created. IMO, providing this link adds value at no cost. If you feel strongly I can remove it, but I like having it here.